tests/benchmarks/data_structure_perf/rbtree_perf/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  benchmark.data_structures:
+    tags: rbtree req-test
```

Comment:
not sure what req-test is, please do not add arbitrary tags that are difficult to understand

Reply:
Hi @nashif , I have re-pushed the PR to modify the tags and make them more clear. PR link: https://github.com/zephyrproject-rtos/zephyr/pull/28110